scripts/addheaders.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
may be better to use `#!/usr/bin/env bash` here for better/wider compatibility

https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash

Reply:
indeed